draft-ietf-quic-transport.md

```diff
@@ -1503,7 +1548,7 @@ An endpoint MUST NOT send a parameter more than once in a given transport
 parameters extension.  An endpoint SHOULD treat receipt of duplicate transport
 parameters as a connection error of type TRANSPORT_PARAMETER_ERROR.
 
-A server MUST include the original_connection_id transport parameter
+A server MUST include the original_destination_connection_id transport parameter
```

Comment:
This is inconsistent with the error condition above:

> * absence of the original_destination_connection_id transport parameter from the server"

Reply:
Not following.  These are consistent by my reading:

> A server MUST include the original_destination_connection_id transport parameter

> An endpoint MUST
treat any of the following as a connection error of type PROTOCOL_VIOLATION:
> * absence of the original_destination_connection_id transport parameter from the server

Reply:
@martinthomson you truncated the sentence in your quote. The full sentence from your PR is:

> A server MUST include the original_destination_connection_id transport parameter
> ({{transport-parameter-definitions}}) if it sent a Retry packet to enable
> validation of the Retry, as described in {{packet-retry}}.

How about we keep the first line, but replace the rest:

> A server MUST include the original_destination_connection_id transport parameter
> to mitigate an attacker's ability to tamper with connection IDs during the handshake.

Reply:
Ahh, that's lazy of me.  Yes, that is good.  Thanks.

Reply:
Oh, that's a throwback paragraph.  I'm going to replace that with a more generic reference.